purchases/src/test/java/com/revenuecat/purchases/backend_integration_tests/ProductionBackendIntegrationTest.kt

```diff
@@ -198,4 +200,34 @@ internal class ProductionBackendIntegrationTest: BaseBackendIntegrationTest() {
         verify(exactly = 1) { sharedPreferencesEditor.apply() }
         assertSigningNotPerformed()
     }
+
+    @Test
+    fun `can get customer center config data`() {
+        var customerCenterConfigData: CustomerCenterConfigData? = null
+        ensureBlockFinishes { latch ->
+            backend.getCustomerCenterConfig(
+                appUserID = "test-user-id",
+                onSuccessHandler = { receivedData ->
+                    customerCenterConfigData = receivedData
+                    latch.countDown()
+                },
+                onErrorHandler = { error ->
+                    fail("Expected success. Got $error")
+                }
+            )
+        }
+        if (customerCenterConfigData == null) {
+            error("Expected customer center config data")
+        }
+        customerCenterConfigData?.let { customerCenterConfigData->
```

Comment:
There is a lint warning here about shadowing the `customerCenterConfigData` name.